pkg/fs/confirmeddir.go

```diff
@@ -26,15 +26,17 @@ import (
 // that was confirmed to point to an existing directory.
 type ConfirmedDir string
 
-// Return a temporary dir, else error.
+// NewTmpConfirmedDir returns a temporary dir, else error.
 // The directory is cleaned, no symlinks, etc. so its
 // returned as a ConfirmedDir.
 func NewTmpConfirmedDir() (ConfirmedDir, error) {
 	n, err := ioutil.TempDir("", "kustomize-")
 	if err != nil {
 		return "", err
 	}
-	return ConfirmedDir(n), nil
+	deLinked, err := filepath.EvalSymlinks(n)
```

Comment:
Please add a comment here explaining the need for this.
With no context, it doesn't make sense.

Reply:
Thanks for fixing btw.  I don't have a mac to repro this.

Reply:
> Please add a comment here explaining the need for this.
> With no context, it doesn't make sense.

added

> Thanks for fixing btw. I don't have a mac to repro this.
my pleasure